src/main/java/se/sundsvall/digitalregisteredletter/service/mapper/AttachmentMapper.java

```diff
@@ -18,12 +19,10 @@ public AttachmentMapper(final BlobUtil blobUtil) {
 		this.blobUtil = blobUtil;
 	}
 
-	public List<AttachmentEntity> toAttachmentEntities(final Attachments attachments) {
-		return Optional.ofNullable(attachments).map(Attachments::files)
-			.map(files -> files.stream()
-				.map(this::toAttachmentEntity)
-				.collect(Collectors.toList())) // Mutable list
-			.orElse(null);
+	public List<AttachmentEntity> toAttachmentEntities(final List<MultipartFile> attachments) {
+		return Optional.ofNullable(attachments).orElse(emptyList()).stream()
+			.map(this::toAttachmentEntity)
+			.collect(Collectors.toList());// Mutable list
```

Comment:
Hm, according to documentation:
```
Returns a Collector that accumulates the input elements into a new List. There are no guarantees on the type,
mutability, serializability, or thread-safety of the List returned; if more control over the returned List is required,
use [toCollection(Supplier)]
```
the Collector.toList does not guarantee mutability. Use `Collectors.toCollection(ArrayList::new)` instead?